examples/src/main/scala/sttp/tapir/examples/BasicAuthenticationServer.scala

```diff
@@ -0,0 +1,41 @@
+package sttp.tapir.examples
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+import sttp.client3._
+import sttp.model.StatusCode
+import sttp.tapir.EndpointInput.WWWAuthenticate
+import sttp.tapir._
+import sttp.tapir.model._
+import sttp.tapir.server.akkahttp._
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+object BasicAuthenticationServer extends App {
```

Comment:
all examples have the server interpreter impl in their name, so this should be `BasicAuthenticationAkkaServer`. Also, please add a link in `examples.md`